src/modal_analysis.cu

```diff
@@ -396,9 +448,9 @@ void MODAL_ANALYSIS::preprocess(char *input_dir, Atom *atom)
         }
 
         size_t eig_size = sizeof(float) * num_participating * num_modes;
-        CHECK(cudaMallocManaged((void **)&eig_x,eig_size));
-        CHECK(cudaMallocManaged((void **)&eig_y,eig_size));
-        CHECK(cudaMallocManaged((void **)&eig_z,eig_size));
+        eigx.resize(eig_size, Memory_Type::managed);
```

Comment:
Here, eig_size is the memory. Should use num_participating * num_modes instead. Otherwise more memory than needed is allocated. This can be a reason for slow-down.

Reply:
Good catch! That was a mistake. Updating that alone makes the code ~1% slower than #55.